examples/core/src/main/scala/com/twitter/util/CountDownLatch.scala

```diff
@@ -3,13 +3,13 @@ package com.twitter.util
 import java.util.concurrent.TimeUnit
 
 class CountDownLatch(val initialCount: Int) {
-  val underlying = new java.util.concurrent.CountDownLatch(initialCount)
-  def count = underlying.getCount
-  def isZero = count == 0
-  def countDown() = underlying.countDown()
-  def await() = underlying.await()
-  def await(timeout: Duration) = underlying.await(timeout.inMillis, TimeUnit.MILLISECONDS)
-  def within(timeout: Duration) = await(timeout) || {
+  val underlying: java.util.concurrent.CountDownLatch = new java.util.concurrent.CountDownLatch(initialCount)
```

Comment:
Why hasn't this been imported?

Reply:
CountDownLatch is in scope, scala.concurrent is in scope, and util is the package name. Hilariously, it seems that the fully qualified name is necessary.

Reply:
https://github.com/twitter/rsc/issues/319